src/unstract/sdk/exceptions.py

```diff
@@ -39,3 +39,11 @@ class OCRError(SdkError):
 
 class RateLimitError(SdkError):
     DEFAULT_MESSAGE = "Running into rate limit errors, please try again later"
+
+
+class FileStorageError(SdkError):
+    pass
```

Comment:
Adding a default message here would be beter to avoid it return `Somwthing went wrong`

Reply:
Done